core/ir/opnd_api.h

```diff
@@ -1858,6 +1872,10 @@ struct _opnd_t {
             byte /*bool*/ scaled : 1;
             /* Shift offset amount */
             byte /*uint*/ scaled_value : 3;
+            /* Indicates the element size for vector base and index registers.
+             * This is ignored if the base and index registers are scalar registers.
```

Comment:
The reader will not understand how 1 bit can indicate the size: please point at #ELEMENT_SIZE_SINGLE and its counterpart.